const throwError = () => {
  throw new Error("This is my error updated!!!");
};

export default throwError;
